l10n_fr_mis_reports/README.rst

```diff
@@ -0,0 +1,56 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
```

Comment:
In the lastest (moving target :wink: README.rst template) I think there is a link target.
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst